File: resolucoes/java/lista01/src/main/java/org/example/Ex019.java
/*Crie um algoritmo que leia o nome e as duas notas de um aluno,
 calcule a sua média e mostre na tela. No final, analise a média
e mostre se o aluno teve ou não um bom aproveitamento (se ficou acima da média 7.0). */

package org.example;

import java.util.Scanner;

public class Ex019 {
    public static void main(String[] args) {
        Ex019.main();
    }

    public static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite sua a primeira nota: ");
        Float nota1 = sc.nextFloat();
        System.out.print("Digite sua a segunda nota: ");
        Float nota2 = sc.nextFloat();
        float media = (nota1 + nota2) / 2;
        if (media >= 7) {
            System.out.printf("Sua média foi %s,houve um bom aproveitamento",media);
        } else {
            System.out.printf("Sua média foi %s,houve um mal aproveitamento",media);
        }
        sc.close();

    }
}

File: resolucoes/JavaScript/lista01/ex019.js
const prompt = require('prompt-sync')();

function main() {

const nome = prompt("Qual seu nome?")
const nota1 = Number(prompt("Qual foi sua primeira nota?"))
const nota2 = Number(prompt("Qual foi sua segunda nota?"))

const media = (nota1 * nota2) / 2

if (media >= 7.0) {
    console.log(`${nome} teve média de ${media.toFixed(1)} e teve um bom aproveitamento.`)
} else {
    console.log(`${nome} teve média de ${media.toFixed(1)} e precisa melhor seu desempenho.`)
}
}

module.exports = {
    main
}